dockertest/subtest.py

```diff
@@ -429,13 +458,61 @@ def try_all_stages(self, name, subsubtest):
                               self.exception_info["error_source"],
                               detail)
             exc_info = self.exception_info["exc_info"]
+            # Ignore known problems in specific docker builds
+            if self.is_known_failure(name):
+                self.logwarning("Treating %s subsubtest failure as PASS", name)
+                self.final_subsubtests.add(name)
             # Treat N/A as passed, even though this may hide obscure failures
             if isinstance(detail, TestNAError):
```

Comment:
I don't know what possible circumstance could trigger this, because the workflow I envision is:

1. test fails in ADEPT
2. human looks at it, files a bz (or recognizes an existing one), adds exception to known-b0rked file

Raising `TestNAError` from within a subtest is when we have situations that are broadly tuned to a docker major version (1.12, 1.13, etc); exceptions are for specific builds and, at times, minor versions (1.12.5).

That said, let's pretend that we do get a `TestNAError` that also matches a known exception. In that case, the outcome is that there will be two warning messages. The outcome is the same (test is marked as passed). I can live with duplicate warnings because this is a situation that just shouldn't happen.

Does that address your question?